templates/router.js

```diff
@@ -12,6 +12,11 @@ router.get('/', {controllerName}.list);
  */
 router.get('/:id', {controllerName}.show);
 
+/*
+ * GET
+ */
+router.get('/p/paginate', {controllerName}.paginate);
```

Comment:
Why use the prefix P?
It should be in the getAll with a condition if there are pagination queryParams.

Reply:
Removed. Pagination for another PR